Dockerfile

```diff
@@ -25,11 +32,17 @@ RUN cmake -S . \
     -DBUILD_EXAMPLES=OFF \
     -DBUILD_TESTS=OFF \
     -DBUILD_DOCS=OFF \
-    -DDISABLE_AWS_IOMANAGER=ON \
+    -DDISABLE_STRICT_WARNINGS=OFF \
     -DDISABLE_AZURESDKFORCPP_IOMANAGER=OFF \
+    -DDISABLE_AWS_IOMANAGER=ON \
```

Comment:
It is nitpicking. Can we reduce the change by moving this line to the top.

Reply:
It was done on purpose.
Before the thing we are interested in (azure) was hidden in between stuff that we turned off.
Now I moved it up so that first we
- turn off general build stuff not interesting for us
- turn on Azure
- turn off other IO we don't care about
- turn on/off build of libraries that we need